docker_interop/app/src/main/java/tech/pegasys/qbft/interop/QbftDockerComposeApp.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package tech.pegasys.qbft.interop;
+
+import org.hyperledger.besu.consensus.qbft.QbftExtraDataCodec;
+import org.hyperledger.besu.crypto.KeyPair;
+import org.hyperledger.besu.crypto.SECP256K1;
+import org.hyperledger.besu.ethereum.core.Address;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static tech.pegasys.qbft.interop.DockerComposeYaml.IP_PREFIX;
+
+@Command(name = "qbft-docker-compose", mixinStandardHelpOptions = true, version = "qbft-docker-compose 1.0",
+        description = "Generate docker-compose.yml and required files to run Besu and GoQuorum in QBFT interop mode.")
+public class QbftDockerComposeApp implements Callable<Integer> {
+    private static final SECP256K1 secp256K1 = new SECP256K1();
+
+    @CommandLine.Option(names = {"-b", "--besuNodes"}, description = "Number of Besu Nodes to include in docker-compose. Default: ${DEFAULT-VALUE}")
+    private int besuNodes = 2;
+    @CommandLine.Option(names = {"-q", "--quorumNodes"}, description = "Number of Quorum Nodes to include in docker-compose. Default: ${DEFAULT-VALUE}")
+    private int quorumNodes = 2;
+    @CommandLine.Option(names = {"-d", "--destination"}, description = "destination directory where docker-compose will be generated. Default: ${DEFAULT-VALUE}")
+    private File destination = Path.of("out").toFile().getAbsoluteFile();
```

Comment:
How can the round timeout and block time be configured? they don't need to be a command line option could be just something in the env file or some template file if that's easier. but we do need to be able to adjust these values for our testing.

Reply:
options introduced. They can also be modified in `app/.../resources/` in template files directly as well.